### PR TITLE
Update to add layer tree and some more properties

### PR DESCRIPTION
**Pull request by matthijsln**
HTM-247

### CODE REVIEW

File: src/main/resources/tailormap-api.yaml [hunk 1]
```diff
@@ -66,13 +66,36 @@ components:
...
           type: array
           items:
             $ref: './model.yaml#/components/schemas/AppLayer'
         crs:
           $ref: './model.yaml#/components/schemas/CoordinateReferenceSystem'
-
+      required:
+        - initialExtent
+        - services
+        - noBaseLayerOption
```
Comment (geertplaisier) [inline src/main/resources/tailormap-api.yaml:94]: Remove this

File: src/main/resources/tailormap-api.yaml [hunk 2]
```diff
@@ -66,13 +66,36 @@ components:
...
+        - rootBaseLayerGroup
```
Comment (geertplaisier) [inline src/main/resources/tailormap-api.yaml:95]: Rename these

**Review by geertplaisier — CHANGES_REQUESTED** [review]

File: src/main/resources/model.yaml [hunk 3]
```diff
@@ -103,29 +103,62 @@ components:
...
+          description: Only a single node in a tree will have this set to true.
+          type: boolean
+        name:
+          description: Display name of this tree node. If appLayerId is present, this is the title of that AppLayer.
+          type: string
+        childrenIds:
+          description: Ordered list of ids of LayerTreeNodes that are children of this node.
+          type: array
+          items:
+            type: integer
```
Comment (geertplaisier) [inline src/main/resources/model.yaml:157]: This type should be string, it references a LayerTreeNode id

**Review by geertplaisier — CHANGES_REQUESTED** [review]

**Review by geertplaisier — APPROVED** [review]

**Review by mprins — APPROVED** [review]
LGTM

**Comment by mprins** [comment]
I think this should be merged into https://github.com/B3Partners/tailormap-api/pull/83 (if still needed and closed - it seems https://github.com/B3Partners/tailormap-api/commits/HTM-248-implement-api has all the commits of this branch + more)

**Comment by matthijsln** [comment]
See #86.

**Comment by mprins** [comment]
duhh, I meant #86...

**Pull request closed without merging**